src/plugins/vis_type_vislib/opensearch_dashboards.json

```diff
@@ -3,7 +3,7 @@
   "version": "opensearchDashboards",
   "server": true,
   "ui": true,
-  "requiredPlugins": ["charts", "data", "expressions", "visualizations", "opensearchDashboardsLegacy", "visTypeVega"],
+  "requiredPlugins": ["charts", "data", "expressions", "visualizations", "opensearchDashboardsLegacy", "visTypeVega", "visAugmenter"],
   "optionalPlugins": ["visTypeXy"],
   "requiredBundles": ["opensearchDashboardsUtils", "visDefaultEditor"]
```

Comment:
If the helper fn src code is the only part needed from `visAugmenter`, prefer to add into `requiredBundles` instead of `requiredPlugins`. `requiredPlugins` is more reserved for accessing services needed from the parent plugin when running the `setup()`/`start()` lifecycle fns.

Reply:
done